plugins/dell-kestrel/fu-dell-kestrel-plugin.c

```diff
@@ -408,6 +408,36 @@ fu_dell_kestrel_plugin_modify_config(FuPlugin *plugin,
 	return fu_plugin_set_config_value(plugin, key, value, error);
 }
 
+static gboolean
+fu_dell_kestrel_plugin_backend_device_removed(FuPlugin *plugin, FuDevice *device, GError **error)
+{
+	const gchar *cache_keys[] = {"ec", "mst", "usb4", NULL};
+
+	if (FU_IS_USB_DEVICE(device)) {
+		guint16 vid = fu_device_get_vid(device);
+		guint16 pid = fu_device_get_pid(device);
+		g_autofree gchar *key = g_strdup_printf("USB\\VID_%04X&PID_%04X", vid, pid);
+
+		if (fu_plugin_cache_lookup(plugin, key)) {
+			fu_plugin_cache_remove(plugin, key);
+			g_debug("Removed %s (%s) from plugin cache",
+				fu_device_get_name(device),
+				fu_device_get_id(device));
+		}
+	}
+
+	for (size_t i = 0; cache_keys[i] != NULL; i++) {
+		if (fu_plugin_cache_lookup(plugin, cache_keys[i])) {
+			fu_plugin_cache_remove(plugin, cache_keys[i]);
+			g_debug("Removed %s (%s) from plugin cache",
```

Comment:
lowercase `removed` pls -- also that debug could probably be moved into fu_plugin_cache_remove() itself..

Reply:
Fixed, thanks.